client/my-sites/sidebar/index.jsx

```diff
@@ -407,7 +415,9 @@ export class MySitesSidebar extends Component {
 					<span className="menu-link-text" data-e2e-sidebar="Plan">
 						{ translate( 'Plan', { context: 'noun' } ) }
 					</span>
-					<span className="sidebar__menu-link-secondary-text">{ planName }</span>
+					{ ! ( isJetpack && ! isAtomicSite && ! isVip ) && (
```

Comment:
This condition is quite complex. Could we maybe move it to a separate constant like `hasPlanName` and add a comment to explain when do we want to show or hide it?

Reply:
Thanks, done in https://github.com/Automattic/wp-calypso/pull/37959/commits/bb917a9c6ee103c0f7ecc35b3f542d1b26a8a701.